spec/vcr_cassettes/Alert/Triggers/Templates/setup.yml

```diff
@@ -37,7 +37,7 @@ http_interactions:
       Pragma:
       - no-cache
       Date:
-      - Tue, 27 Sep 2016 18:17:14 GMT
+      - Fri, 07 Oct 2016 15:36:32 GMT
```

Comment:
Yes, don't re-record if they are not changed. But I fear that is harder done than said. But such "empty" recordings could be removed before committing